NICE.Identity.Management/ClientApp/src/components/App/App.tsx

```diff
@@ -18,21 +18,61 @@ import { SelectRoles } from "./../../views/SelectRoles/SelectRoles";
 import { EditUser } from "./../../views/EditUser/EditUser";
 import { DeleteOrganisation } from "./../../views/DeleteOrganisation/DeleteOrganisation";
 
+import { Endpoints } from "../../data/endpoints";
+import { fetchData } from "../../helpers/fetchData";
+import { isDataError } from "../../helpers/isDataError";
+
+export type MyAccountDetails = {
+	displayName: string,
+	links: []
+};
+
 export class App extends React.Component {
-	render(): JSX.Element {
+	state = {
+		isLoading: false,
+		auth: {} as IdamProviderProps
+	}
+	async componentDidMount(): Promise<void> {
+		const fetchOptions = {
+			method: "GET",
+			headers: { "Content-Type": "application/json" }
+		};
```

Comment:
```suggestion
               this.setState({ isLoading: true });
               
		const fetchOptions = {
			method: "GET",
			headers: { "Content-Type": "application/json" }
		};
```

Move the isLoading: true from below up here.